certora/specs/Health.spec

```diff
@@ -75,6 +78,44 @@ filtered {
     assert !priceChanged => stillHealthy;
 }
 
+// The general case of liquidate for the stayHealthy rule
+//   - the user checked is not the borrower,
+//   - the market of the liquidation is the market of the user,
+//   - there is still some borrow on the market after liquidation,
+//   - there was no bad debt realization.
```

Comment:
it times out with bad debt realisation?

Reply:
Yes, it times out with bad debt realization. We tried long and hard and gave up in the end